src/popup/router/components/PendingTxs.vue

```diff
@@ -10,44 +10,21 @@
 
 <script>
 import { mapState } from 'vuex';
-import { setInterval, clearInterval } from 'timers';
 import TransactionItem from './TransactionItem';
 
 export default {
   components: { TransactionItem },
-  computed: {
-    ...mapState(['transactions']),
-    filteredPendings() {
-      return this.transactions.pending
+  computed: mapState({
+    transactions: ({ transactions: { pending } }) =>
+      pending
         .filter(({ amount, hash }) => !Number.isNaN(+amount) && hash)
-        .map(({ time, hash, tipUrl, ...t }) => ({
+        .map(({ time, hash, tipUrl, ...tx }) => ({
           time,
           hash,
           tipUrl,
-          tx: { ...t },
+          tx,
           pending: true,
-        }));
-    },
-  },
-  created() {
-    const checkMined = setInterval(() => this.checkPendingTxMined(), 2500);
```

Comment:
There is no guarantee that internal pulling will end in 2.5s, so several pullings can be started at the same time.